src/rpc/rpc.cc

```diff
@@ -593,8 +593,10 @@ namespace rpc {
       using header_and_buffer_type = std::tuple<std::optional<uint64_t>, uint64_t, int64_t, opt_buf_type>;
       using return_type = future<header_and_buffer_type>;
       using header_type = std::tuple<std::optional<uint64_t>, uint64_t, int64_t, uint32_t>;
+      static constexpr size_t raw_header_size = sizeof(uint64_t) + sizeof(int64_t) + sizeof(uint32_t);
```

Comment:
While we are at it may be we should document here what each of those is for?

Reply:
Sure. I'll push v2 with more comments